package.json

```diff
@@ -4,8 +4,8 @@
   "license": "W3C",
   "description": "A technical specification pre-processor.",
   "engines": {
-    "node": ">=10",
-    "npm": ">=5"
+    "node": ">=14",
+    "npm": ">=6"
```

Comment:
Should we use npm 7 now? Seems mainstream now.

Reply:
Yeah, and this would be the breaking change.